File: examples/QuickStart/Assets/Fllow.cs
using UnityEngine;
using System.Collections;

  public class Fllow : MonoBehaviour
  {
    public Transform player; // 主角位置
    public float speed = 5f; // 相机速度   
    Vector3 distance; // 主角和摄像机之间的距离                   


    void Start()
    {
     
      distance = transform.position - player.position;
    }

    void Update()
    {
      Vector3 targetCamPos = player.position + distance;

      transform.position = targetCamPos;
    }
  }

File: examples/QuickStartMiniGameProject/minigame/__temp/__scripts/src/Fllow.js
Bridge.assembly("minigame-adaptor-project", function ($asm, globals) {
    "use strict";

    Bridge.define("Fllow", {
        inherits: [MiniGameAdaptor.MonoBehaviour],
        fields: {
            player: null,
            speed: 0,
            distance: null
        },
        ctors: {
            init: function () {
                this.distance = new MiniGameAdaptor.Vector3();
                this.speed = 5.0;
            }
        },
        methods: {
            Start: function () {

                this.distance = MiniGameAdaptor.Vector3.op_Subtraction(this.transform.position.$clone(), this.player.position.$clone());
            },
            Update: function () {
                var targetCamPos = MiniGameAdaptor.Vector3.op_Addition(this.player.position.$clone(), this.distance.$clone());

                this.transform.position = targetCamPos.$clone();
            }
        }
    });
});
